app/src/main/java/com/ece493/group5/adjustableaudio/models/EqualizerModel.java

```diff
@@ -1,64 +1,174 @@
 package com.ece493.group5.adjustableaudio.models;
 
-import java.lang.reflect.Array;
+
+import android.content.Context;
+
+import com.ece493.group5.adjustableaudio.storage.SaveController;
+
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
 
 public class EqualizerModel
 {
+    private static final String TAG = EqualizerModel.class.getSimpleName();
+
     List<EqualizerPreset> equalizerPresets;
+    int currentEqualizerSettingPosition;
+    HashMap<Integer, Integer> currentEqualizerBandValues;
+    int currentLeftVolume;
+    int currentRightVolume;
+    String currentEqualizerName;
 
```

Comment:
perhaps it might be possible to replace the variables:
```java
    HashMap<Integer, Integer> currentEqualizerBandValues;
    int currentLeftVolume;
    int currentRightVolume;
    String currentEqualizerName;
```
with:
```java
EqualizerPreset currentPreset;
```

which is cleaner in my opinion.